### PR TITLE
add schema and some explaining text

### PR DESCRIPTION
**Pull request by jbcr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in tree parts (read, one or more step, one or more writer).
```
Comment (jeremycr) [inline README.md]:
```suggestion
Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).
```

File: README.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
...
+
+The reader can read data from everywhere and return data row by row. Each step processes the current row data.
```
Comment (jeremycr) [inline README.md]:
```suggestion
The reader can read data from anywhere and return data row by row. Each step processes the current row data. 
```

File: README.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in tree parts (read, one or more step, one or more writer).
+
+The reader can read data from everywhere and return data row by row. Each step processes the current row data. 
+The steps order is defined by the step adding into the dataflow builder.
```
Comment (jeremycr) [inline README.md]: The steps are executed in the order in which they are added.

File: README.md [hunk 4]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in tree parts (read, one or more step, one or more writer).
+
+The reader can read data from everywhere and return data row by row. Each step processes the current row data. 
+The steps order is defined by the step adding into the dataflow builder. 
+And finally, one or more writers save the row to everywhere.
```
Comment (jeremycr) [inline README.md]:
```suggestion
And finally, one or more writers save the row anywhere you want.
```

File: README.md [hunk 5]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
...
+
+Like this schema show, you can define more than one dataflow:
```
Comment (jeremycr) [inline README.md]:
```suggestion
Like shown in this schema, you can define more than one dataflow:
```

File: README.md [hunk 6]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).
+
+The reader can read data from anywhere and return data row by row. Each step processes the current row data. 
+The steps are executed in the order in which they are added.
+And finally, one or more writers save the row anywhere you want.
+
+Like shown in this schema, you can define more than one dataflow:
```
Comment (SofLesc) [inline README.md]:
```suggestion
As the following schema shows, you can define more than one dataflow:
```

File: README.md [hunk 7]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).
```
Comment (SofLesc) [inline README.md]: Dataflow uses a linear generic workflow in three parts:
- a reader
-  any number of steps
- one or more writers).

File: README.md [hunk 8]
```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
...
+
+The reader can read data from anywhere and return data row by row. Each step processes the current row data. 
+The steps are executed in the order in which they are added.
+And finally, one or more writers save the row anywhere you want.
```
Comment (SofLesc) [inline README.md]: It does not really make sense to say `finally` if the is not a first step.
Change this into something like : first the reader ... then ... finally ...
Or remove the `finally`

**Pull request merged**